newIDE/app/src/ObjectsRendering/PixiResourcesLoader.js

```diff
@@ -232,6 +234,72 @@ export default class PixiResourcesLoader {
     return material;
   }
 
+  /**
+   * Return the three.js material associated to the specified resource name.
+   * @param project The project
+   * @param resourceName The name of the resource
+   * @param options
+   * @returns The requested material.
+   */
+  static get3DModel(
+    project: gdProject,
+    resourceName: string
+  ): Promise<THREE.Object3D> {
+    const loaded3DModel = loaded3DModels[resourceName];
+    if (loaded3DModel) return Promise.resolve(loaded3DModel);
+
+    if (!project.getResourcesManager().hasResource(resourceName))
+      return invalidTexture;
+
+    const resource = project.getResourcesManager().getResource(resourceName);
+    if (resource.getKind() !== 'model3D') return invalidTexture;
+
+    const url = ResourcesLoader.getResourceFullUrl(project, resourceName, {
+      isResourceForPixi: false,
```

Comment:
I wonder if this should be true, and renamed to `isResourceForSceneEditor`.
This is because these URLs will be loaded by a fetch/XMLHttpRequest and could be cached differently compared to something loaded and shown by the browser... but this distinction only applies to `img` (if they try to load the same URL as what is loaded by PixiJS with fetch/XMLHttpRequest, there is a risk of caching the wrong headers).